scripts/plugins/githubLinks.js

```diff
@@ -17,10 +17,17 @@ function githubLinks(options) {
       const file = files[path];
       path = path.replace('.html', '.md').replace(/\\/g, '/');
       const url = `https://github.com/nodejs/nodejs.org/edit/main/locale/${options.locale}/${path}`;
-
-      const contents =
-        file.contents.toString() +
-        ` <input type = "hidden" id = "editOnGitHubUrl" value="${url}"/> `;
+      const editOnGitHubTrans = options.site.editOnGithub || 'Edit on GitHub';
+      const contents = file.contents
+        .toString()
+        .replace(
+          /<div class="openjsfoundation-footer-edit"><\/div>/,
+          (match, $1, $2) => {
+            return `<div class="openjsfoundation-footer-edit">
```

Comment:
Could we put the callback somewhere else? Like separate `const`?

Reply:
OK，it's done.